simple-backend/nlpviewer_backend/handlers/nlp.py

```diff
@@ -135,9 +67,10 @@ def load_model(request, model_name: str):
 
 
 @require_login
-def run_pipeline(request, document_id, model_name):
+def run_pipeline(request, document_id):
   doc = Document.objects.get(pk=document_id)
   docJson = model_to_dict(doc)
+  model_name = doc.project.name
```

Comment:
Assigning `project name` to `model_name` is confusing, probably we should rename and make the logic clear.